docs/source/quick_start.rst

```diff
@@ -0,0 +1,199 @@
+============
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users.  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+-----------------------------
+Download Latest, Set Up Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" will need to be updated for each release 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/tag/4.2.01>`
+
+
+::
+
+  unzip kokkos-x.y.z.zip (or tar -xzf kokkos-x.y.z.tar.gz)
+  cd kokkos-x.y.z
+  mkdir build
+  cd build
+
+
+-------------------
+Basic Configuration
+-------------------
+
+.. note::
+
+  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
+
+
+Serial (CPU)
+~~~~~~~~~~~~
+::
+
+  cmake \
+    -DKokkos_ARCH_NATIVE=ON \
+    -DKokkos_ENABLE_OPENMP=OFF \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+.. note::
+
+  Kokkos will attempt to autodetect GPU microarchitecture, but it is also possible to specify the desired `GPU architecture <https://kokkos.org/kokkos-core-wiki/keywords.html#gpu-architectures>`_.   
+
+OpenMP (CPU Parallelism)
+~~~~~~~~~~~~~~~~~~~~~~~~
+::
+
+  cmake \
+   -DKokkos_ARCH_NATIVE=ON \
+   -DKokkos_ENABLE_OPENMP=ON \
+   -DKokkos_ENABLE_SERIAL=ON \
+   -DCMAKE_BUILD_TYPE=Release ..
+
+
+CUDA (CPU and GPU)
+~~~~~~~~~~~~~~~~~~
+
+::
+
+  cmake \
+    -DKokkos_ARCH_NATIVE=ON \
+    -DKokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE=OFF \
+    -DKokkos_ENABLE_CUDA=ON \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
```suggestion
  cmake -S . -B -DKokkos_ENABLE_CUDA=ON -DCMAKE_BUILD_TYPE=Release
```

BTW if nvcc is not in their path this will fail (but I think most clusters will load cuda as an environment module)

Reply:
I prefer the current version.

Reply:
I missed a `-B build` in there but regardless, if you prefer the `cmake ..` approach (I'm not hard set either way), I still think it's important to remove the ARCH_NATIVE, the CUDA_RELOCATABLE_DEVICE_CODE and the ENABLE_SERIAL flags as they are not required for building.